cypress/integration/modal.spec.ts

```diff
@@ -0,0 +1,39 @@
+import * as commands from "../support/commands";
+
+context("modal", () => {
+  beforeEach(() => {
+    commands.resetProxyState();
+    commands.onboardUser();
+    cy.visit("./public/index.html");
+  });
+
+  it("no modal stacking possible", () => {
+    cy.log(
+      "click the New Project button and check the corresponding modal is opened."
+    );
+    commands.pick("new-project-button").click();
+    commands.pick("modal", "create-project").should("exist");
+
+    cy.log(
+      "now open the shortcuts modal and verify that it is the only modal open"
+    );
+    cy.get("body").type("?");
+    commands.pick("modal", "hotkey-modal").should("exist");
+    commands.pick("modal", "create-project").should("not.exist");
+  });
+
+  it("navigating to the Settings screen closes any open modal", () => {
```

Comment:
I'd wrap this in a context, like so:

```suggestion
context("when navigating to the settings screen", () => {
  it("closes any open modal", () => {
```

Reply:
Done :heavy_check_mark: